tabpy-tools/tools_tests/test_client.py

```diff
@@ -8,14 +7,12 @@
 
 from tabpy_tools.client import Client
 
-import requests
-
 
 class TestClient(unittest.TestCase):
 
     def setUp(self):
         self.client = Client("http://example.com/")
-        self.client._service = Mock()
+        self.client._service = Mock()  # TODO: should spec this
```

Comment:
Because this isn't built to a predefined spec it makes very loose assertions. There's a typo in one of the tests that dangerously gets assigned as a property of the Mock instead of actually making an assertion

Reply:
Follow up is #214